.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,38 @@
+name: E4E Packaging Workflow
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+        # order operating systems from best to worst
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get E4E package version
+        id: npm-version
+        uses: nyaa8/package-version@v1.0.3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install Dependencies
```

Comment:
I believe, that we should use yarn, tbh.. Do you think we need to use npm?

Reply:
May I ask you to change `ci` script to `lint`, `build` and `test` steps from our `package.json` here?

I believe, that CI pipeline should just call npm scripts and do not perform any other actions, which will give us ability to use the same scripts for different CI runners.